client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -33,7 +33,7 @@ export default function BoostCalibration({
   );
 
   const handleSubmit = useCallback(
-    (event: React.FormEvent<HTMLInputElement>) => {
+    (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
And this should be `HTMLFormElement` as that's the expected type for the form's `onSubmit`.